### DISCUSSION TITLE
Review Code of Conduct - Discussion

### DISCUSSION THREAD

**Discussion by stephen-rowan** [NFT-DAO/Governance-HOLON#46] [Open-Source]
# Review, amend and update Code of Conduct

## Review, amend and update Code of Conduct for NFT-DAO Governance HOLON GitHub Repository.

**This Discussion was converted from an Issue to serve as a discussion thread for this project.**

**When assigning Issues related to this discussion :**

1. Add the Following Project Categories - [Code Of Conduct](https://github.com/NFT-DAO/Governance-HOLON/projects/6) and [NFT DAO Fund 3 Deliverables](https://github.com/NFT-DAO/Governance-HOLON/projects/6).
1. Add the Following Milestone - [Sign-Off Code of Conduct](https://github.com/NFT-DAO/Governance-HOLON/milestone/6)

## Project Board

https://github.com/NFT-DAO/Governance-HOLON/projects/6

## GitHub Location 

https://github.com/NFT-DAO/Governance-HOLON/blob/main/CODE_OF_CONDUCT.md

## Discord Channel

TBA

**Comment by DanM3rcurius** [comment]
Looks very clear and well written. Checks many correct boxes!
Under [Enforcement Responsibilities](https://github.com/NFT-DAO/Governance-HOLON/blob/fb05d6a274686f6c28c1ae0133ff4ce80f75842a/CODE_OF_CONDUCT.md#enforcement-responsibilities) only "the community leaders are responsible for clarifying and enforcing our standards" - why not each community member? Or what defines a community leader?
Also not too sure about "permanent bans" - can they be revoked by examining good/valuable behaviour? Can we use a different terminology? Wouldn't a long temporary ban be enough? 
Banning leads to polarisation of thoughts outside an environment were they can be reflected upon. Also I find it questionable to give one person or a small entity decision power over a ban from the community, shouldn't the community decide?

🙏

**Comment by stephen-rowan** [comment]
Example of simple Code of Conduct : https://exercism.io/code-of-conduct

**Comment by stephen-rowan** [comment]
> 
> 
> Looks very clear and well written. Checks many correct boxes!
> Under [Enforcement Responsibilities](https://github.com/NFT-DAO/Governance-HOLON/blob/fb05d6a274686f6c28c1ae0133ff4ce80f75842a/CODE_OF_CONDUCT.md#enforcement-responsibilities) only "the community leaders are responsible for clarifying and enforcing our standards" - why not each community member? Or what defines a community leader?
> Also not too sure about "permanent bans" - can they be revoked by examining good/valuable behaviour? Can we use a different terminology? Wouldn't a long temporary ban be enough?
> Banning leads to polarisation of thoughts outside an environment were they can be reflected upon. Also I find it questionable to give one person or a small entity decision power over a ban from the community, shouldn't the community decide?
> 
> 🙏

I haven't looked at the Code of Conduct in detail - as I chose a community standard based on the ["Contributor Covenant"](https://www.contributor-covenant.org/). 

At first glance I kind of agree with you that a DAO may find community *leaders* a problematic term. 
Also permanent bans seem a tad draconian - I agree with your point about polarisation. There should always be a path for redemption. 

We really need to open this review to NFT-DAO Discord members.

**Comment by newmindflow** [comment]
Who would grant the authority to a “leader”? On what basis?

I suggest the term “key holon” with a clear explanation of the fact that authority is derived from the quantifiable contributions to the DAO, be it monetary or otherwise. (How will the value of contributions be decided in the case their not monetary is a different topic we must address separately.)

If we’re building a system in which participation will be ultimately based on governance tokens, wouldn’t a total ban basically amount to theft? As it would imply rendering those tokens invalid?
Imagine a case in which someone exchanges $1 million for DAO governance tokens but then proceeds to have actions that all the other members disagree with and are in violation of the code of conduct. 
Issuing a total ban would basically render his tokens useless.

Edit - I'll reply to myself :-) :

In a system in which you could exchange governance tokens for any other type of currency, you would still be able to exchange your tokens back even after a total ban. It would just mean that you’re no longer able to participate in the DAO.
But still… I’m unconvinced on whether or not a total ban would be a good idea.

**Comment by stephen-rowan** [comment]
Sorry for the delay in responding @newmindflow - I have merged a pull request to change "Community Leaders" to "Key Holon (TBA)"

https://github.com/NFT-DAO/Governance-HOLON/pull/42#issue-615600010
